src/datasources/exchange-api/exchange-api.service.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { ExchangeApi } from './exchange-api.service';
+import { ExchangeResult } from '../../domain/exchange/entities/exchange-result.entity';
+import { FiatCodesExchangeResult } from '../../domain/exchange/entities/fiat-codes-result.entity';
+import { FakeConfigurationService } from '../../common/config/__tests__/fake.configuration.service';
+import { mockNetworkService } from '../../datasources/network/__tests__/test.network.module';
+
+const FIATCODES: FiatCodesExchangeResult = {
+  success: true,
+  symbols: {},
+};
+
+const EXCHANGERESULT: ExchangeResult = {
+  success: true,
+  rates: {},
+  base: '',
+};
+
+describe('ExchangeApi', () => {
+  const fakeConfigurationService = new FakeConfigurationService();
+  fakeConfigurationService.set('exchange.baseUri', 'http://exchange.test');
+  fakeConfigurationService.set('exchange.apiKey', 'aaaaaaaabbbbbbb');
```

Comment:
Probably setting this in `beforeAll` is better as we know that this configuration will "run before all the tests". It's similar to what is being done here but with a more "semantic" way for tests.